reana-ui/src/actions.js

```diff
@@ -407,11 +411,11 @@ export function fetchWorkflowRetentionRules(id) {
   };
 }
 
-export function deleteWorkflow(id, workspace = true) {
+export function deleteWorkflow(id, { workspace = true, allRuns = false } = {}) {
   return async (dispatch) => {
     dispatch({ type: WORKFLOW_DELETE_INIT });
     return await client
-      .deleteWorkflow(id, { workspace })
+      .deleteWorkflow(id, { workspace, allRuns })
       .then((resp) => {
         dispatch({ type: WORKFLOW_DELETED, ...resp.data });
```

Comment:
Very minor: maybe the text of the success notification could be different when deleting _all the runs_ of a workflow (like in `r-client`)? Could be something as simple as changing "workflow" to "workflows" in "workflow successfully deleted", or could be a totally different sentence, like `r-client` does. 

Note that if we want to address this, rather than changing `r-ui`, it might be better to change the `message` property returned by `r-w-c` when deleting more than one run.

Reply:
Created issue: https://github.com/reanahub/reana-workflow-controller/issues/531